src/nvim/os_unix.c

```diff
@@ -355,12 +355,12 @@ int mch_expand_wildcards(int num_pat, char_u **pat, int *num_file,
           }
           j++;
         } else if (!intick
-            && ((flags & EW_KEEPDOLLAR) == 0 || pat[i][j] != '$')
-            && vim_strchr(SHELL_SPECIAL, pat[i][j]) != NULL) {
-          // Put a backslash before a special character, but not
-          // when inside ``. And not for $var when EW_KEEPDOLLAR is
-          // set.
-          *p++ = '\\';
+          && ((flags & EW_KEEPDOLLAR) == 0 || pat[i][j] != '$')
```

Comment:
@raichoo I think it means that on the new line, the line should be indented one in from the opening brace on the line above: eg:

```
&& ((flags & EW_KEEPDOLLAR) == 0 || pat[i][j] != '$')
       && vim_strchr(SHELL_SPECIAL, pat[i][j]) != NULL) {
```